### PR TITLE
Avoid accidental changes to version number in task file

### PR DESCRIPTION
**Pull request by pjar**
Hi - thank you for a great tool!

We use after_party a lot and it is super-useful for us. The thing is we sometimes manually create task files and/or we don't spot in-content timestamp mismatch against the file name.

This change helps avoid such mismatches.

### CODE REVIEW

File: lib/generators/task/templates/deploy.txt.erb [hunk 1]
```diff
@@ -7,6 +7,7 @@ namespace :after_party do
 
     # Update task as completed.  If you remove the line below, the task will
     # run with every deploy (or every time you call after_party:run).
-    AfterParty::TaskRecord.create version: '<%= timestamp %>'
```
Comment (BartlomiejSkwira) [inline lib/generators/task/templates/deploy.txt.erb]: So where was the `timestamp` variable set?

Comment (BartlomiejSkwira) [inline lib/generators/task/templates/deploy.txt.erb]: Ahh I see it was the rake task timestamp

File: lib/generators/task/templates/deploy.txt.erb [hunk 2]
```diff
@@ -7,6 +7,7 @@ namespace :after_party do
...
+    AfterParty::TaskRecord
+      .create version: AfterParty::TaskRecorder.new(__FILE__).timestamp
```
Comment (dissolved) [inline lib/generators/task/templates/deploy.txt.erb]: Yours should work, but so would this.
```suggestion
      .create version: __FILE__.to_i.to_s
```

**Comment by theSteveMitchell** [comment]
@pjar would you mind a quick rebase of master to pull in the Travis config and get a green build?  I know it's been sitting here awhile...I hope you're still...alive

**Comment by pjar** [comment]
@theSteveMitchell I'm still alive but I missed your message - sorry and thank you for doing this for me, and for merging this in - I'm glad it looked helpful not only to me :)

**Pull request merged**